frontends/ol-search-ui/assets/learning-resource-drawer.scss

```diff
@@ -96,7 +96,7 @@
       font-weight: bold;
     }
 
-    @include breakpoint(desktop) {
+    @include breakpoint.up(sm) {
```

Comment:
This is the only change that should have any noticeable impact, and it's a mild css improvement.

Now float-right works whenever the drawer is not in mobile mode.